scripts/__init__.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=unused-variable
```

Comment:
There's a high probability this file could just be an empty file and we don't have to manage any of its contents.

Reply:
Since I'm already working on updating the docs with the changes to templater and the new flags I think it makes more sense to either roll these changes into that PR or do them in a follow up after that's done (which will hopefully be Monday)